plugins/analysis-ukrainian/src/main/java/org/elasticsearch/index/analysis/ukrainian/UkrainianAnalyzerProvider.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.elasticsearch.index.analysis.ukrainian;
```

Comment:
Thanks @johtani. I don't mind putting all code into the `o.e.index.analysis` package, I think I followed some other analysis plugin layout. Does anybody else have an opinion on this? Maybe there was a reason for grouping analysis plugin code in sub-packages? I somehow doubt it and will move all the code of this PR into `o.e.index.analysis` if nobody objects.